presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixHdfsInitializer.java

```diff
@@ -13,19 +13,9 @@
  */
 package io.prestosql.plugin.hive.rubix;
 
-import javax.inject.Qualifier;
+import io.prestosql.plugin.hive.ConfigurationInitializer;
 
-import java.lang.annotation.Retention;
-import java.lang.annotation.Target;
-
-import static java.lang.annotation.ElementType.FIELD;
-import static java.lang.annotation.ElementType.METHOD;
-import static java.lang.annotation.ElementType.PARAMETER;
-import static java.lang.annotation.RetentionPolicy.RUNTIME;
-
-@Retention(RUNTIME)
-@Target({FIELD, PARAMETER, METHOD})
-@Qualifier
-public @interface ForRubix
+public interface RubixHdfsInitializer
```

Comment:
One another idea is to use guice multibinder where you could have several independent `ConfigurationInitializer`.